http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1215,8 +1218,11 @@ class GKEInterpreter[F[_]](
           } yield ()
         } else {
           for {
+            startTime <- F.realTimeInstant
```

Comment:
Could you maybe modify the comment below? It sounds more general than just for Galaxy

Reply:
I didn't add the comment..but sure :)